docs/changelog.md

```diff
@@ -1,5 +1,40 @@
 ## Changelog
 
+### v0.13.0 (1/5/2021)
+
+#### Highlights
+- Support a monocular 3D detection method [FCOS3D](https://arxiv.org/abs/2104.10956)
+- Support ScanNet and S3DIS semantic segmentation dataset
+- Support multi-modality visualization
+
+#### Bug Fixes
+
+- Fix Pytorch 1.8 Compilation issue (#404)
```

Comment:
To be more specific: Fix Pytorch 1.8 Compilation issue in the [scatter_points_cuda.cu](...)